packages/tocco-ui-showcase/src/components/Navigation.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import {Scrollspy} from 'react-scrollspy'
+
+const Navigation = props => {
+  const getComponentSections = (categoryName) => {
+    var sections = []
```

Comment:
I personally don't use `var` in ES6. Wherever possible, I use `const` - and if it must be possible to reassign a value, I use `let` (but I try to avoid it).

Is there a reason that you still use `var` (several occurrences)?